### ISSUE TITLE
Followup to Arcade conversion

### ISSUE THREAD

**Issue by JunTaoLuo** [dotnet/aspnetcore#13350]
Follow up items for https://github.com/aspnet/AspNetCore/issues/7280:

* [ ] Arcade does not have support doing a 'dry run' of microbenchmarks projects during the 'Test' phase. See https://github.com/aspnet/BuildTools/blob/803fd176f1ae147aa8af08e49e76ffda7221a4cd/files/KoreBuild/modules/benchmarks/module.targets#L14-L37 for the korebuild implementation
  - Tracked in https://github.com/dotnet/aspnetcore/issues/7886
* [ ] Ensure the servicing attribute is set correctly on assemblies: cref https://github.com/dotnet/arcade/issues/1526
  - Tracked in https://github.com/dotnet/aspnetcore/issues/18930
* [x] Arcade does not have a good way to specify signcheck exclusions https://github.com/dotnet/arcade/issues/2888 (Edit, there seems to be a way to do this in stages so we should be unblocked here)

These probably don't need to be address right away, or at all.
* [x] Need to find a replacement for API check
* [ ] Need to find a replacement for NuGet Package Verifier
  - Tracked in https://github.com/dotnet/aspnetcore/issues/18931

**Comment by Pilchie** [comment]
@aspnet/build Should this still be in the 3.1 milestone?

**Comment by dougbu** [comment]
@Pilchie it should either
1. Not be in a milestone at all because we track it using https://github.com/orgs/aspnet/projects/2
2. In a 3.1.x milestone based on our feeling of urgency. My opinion is this isn't urgent but I also suspect we'll find more bullets (I just checked one off) are already done.

**Comment by JunTaoLuo** [comment]
Tracking action items separately.
